workflows/cluster_splitter.nf

```diff
@@ -43,9 +43,31 @@ include { CUSTOM_DUMPSOFTWAREVERSIONS } from '../modules/nf-core/custom/dumpsoft
 
 workflow CLUSTER_SPLITTER {
 
-    ID_COLUMN = "sample"
+    ID_COLUMN = "sample_name"
```

Comment:
I am wondering if `ID_COLUMN` here should be named `sample` and `ID_COLUMN2` should be named `irida_id`?

One issue I have encountered here is with ArborView:

![image](https://github.com/user-attachments/assets/dfcbfdca-0b6a-405c-8048-e365ac50efb9)

The `sample` in this image is the IRIDA Next identifier and `sample_name` is the sample name from IRIDA. This does make sense. However, since the leaves in the tree are labeled by the sample name as well, colouring these leaves by metadata and filtering the metadata by the leaf names no longer works.

I am wondering if ArborView uses either a column called `sample` to match to the leaf names in the tree, or the first column in the metadata table. @mattheww95 do you have more insight on this?

If ArborView matches a column called `sample`, then renaming the columns as above would fix the issue. If ArborView instead uses the first column to match with leaf labels, then we'd need to switch the order of the columns in the metadata written to the ArborView HTML file so that `sample_name` appears first.

Reply:
From feedback from Matthew, ArborView uses the left-most column as identifiers to match up with leaf labels. So, I don't think we need to rename these columns, just switch the order so that `sample_name` is the left-most column.

Reply:
> However, since the leaves in the tree are labeled by the sample name as well, colouring these leaves by metadata and filtering the metadata by the leaf names no longer works.

I had not really understood how the arborview tree had been used. Just decided to keep the order that the `samplesheet.csv` had. :clown_face: Good catch! 

Bonus point: It corrects for this code block that kind of didn't make sense.
 ```   // Metadata headers:
    metadata_headers = Channel.value(
        tuple(
            ID_COLUMN2, ID_COLUMN, params.metadata_partition_name,
            params.metadata_1_header, params.metadata_2_header,
            params.metadata_3_header, params.metadata_4_header,
            params.metadata_5_header, params.metadata_6_header,
            params.metadata_7_header, params.metadata_8_header)
        )
```

Made the changes here [cca5611](https://github.com/phac-nml/arboratornf/pull/23/commits/cca5611a547a107241205dd8c3b75d0887917419)